src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -55,15 +55,15 @@ public class CodeStyleOptions
         public static readonly PerLanguageOption<CodeStyleOption<bool>> PreferIntrinsicPredefinedTypeKeywordInDeclaration = new PerLanguageOption<CodeStyleOption<bool>>(nameof(CodeStyleOptions), nameof(PreferIntrinsicPredefinedTypeKeywordInDeclaration), defaultValue: TrueWithNoneEnforcement,
             storageLocations: new OptionStorageLocation[]{
                 new EditorConfigStorageLocation("dotnet_style_predefined_type_for_locals_parameters_members"),
-                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferIntrinsicPredefinedTypeKeywordInDeclaration")});
+                new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.PreferIntrinsicPredefinedTypeKeywordInDeclaration.Dev15")});
```

Comment:
I'm happy to change it.  I chose Dev15 because this option will only be persisted to machines with Dev15 or newer.  Code Style is fine though.